modules/client-configuration/pages/supported-features-amazon.adoc

```diff
@@ -0,0 +1,160 @@
+[[supported-features-amazon]]
+= Supported {amazonlinux} Features
+
+
+This table lists the availability of various features on {amazonlinux} clients.
+
+[IMPORTANT]
+====
+The operating system you run on a client is supported by the organization that supplies the operating system.
+{amazonlinux} is supported by Amazon.
+====
+
+The icons in this table indicate:
+
+* {check} the feature is available on both Salt and traditional clients
+* {cross} the feature is not available
+* {question} the feature is under consideration, and may or may not be made available at a later date
+* {trad} the feature is supported only on traditional clients
+* {salt} the feature is supported only on Salt clients
+
+
+[cols="1,1", options="header"]
+.Supported Features on {amazonlinux} Operating Systems
+|===
+
+| Feature
+| {amazonlinux}{nbsp}2
+
+| Client
+| {salt}
+
+| Operating system packages
+| {salt}
+
+| Registration
+| {salt}
+
+| Install packages
+| {salt}
+
+| Apply patches (requires CVE ID)
+| {question}
```

Comment:
Why the question mark? because it's untested? If you can see the errata information, then CVE audit and applying patches should work too.

Reply:
![image](https://user-images.githubusercontent.com/4226070/112624772-558ee500-8e2e-11eb-8177-f71a3fa25dae.png)

@paususe I assume that if I see this, it means the errata information is available, CVE audit will work and applying patches as well, even if I could not test it myself?

Reply:
Fixed